Tests/Common/Statistics/AssetsUnderManagementCapacityManagerTests.cs

```diff
@@ -0,0 +1,304 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+using QuantConnect.Algorithm;
+using QuantConnect.Data.Market;
+using QuantConnect.Interfaces;
+using QuantConnect.Orders;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Securities;
+using QuantConnect.Statistics;
+using QuantConnect.Tests.Engine.DataFeeds;
+using System;
+using System.Linq;
+using Moq;
+using QuantConnect.Brokerages.Backtesting;
+using QuantConnect.Data;
+using QuantConnect.Lean.Engine.TransactionHandlers;
+using QuantConnect.Tests.Engine;
+
+namespace QuantConnect.Tests.Common.Statistics
+{
+    [TestFixture]
+    public class AssetsUnderManagementCapacityManagerTests
```

Comment:
Nice tests! 👍 
I think one test case that is missing, and would be nice to have, is for the same date, the same symbol, two trades and assert it uses the minimum trade capacity of both trades?